style.css

```diff
@@ -0,0 +1,85 @@
+.ul {
+  list-style: none;
+}
+
+.main-section {
+  background-image: url('img/bg.png');
```

Comment:
- Hey, you've used the wrong background image, kindly check the Figma template to see the background image for the mobile version to improve your design